### DISCUSSION TITLE
Import old settings on upgrade?

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#744] [Help!] [answered]
Hello,

Wondering if it's possible to auto-import settings (edge style etc) and keyboard shortcuts on version upgrades? I'm not a programmer so I'm not sure if this goes against good programming habits, but it would definitely be a nice quality of life upgrade. Even the position of the Videos / Labelling Suggestions windows all move.

Best,
Stefan

**Comment by talmo** [comment, marked as answer]
Hi @olinesn,

We do this as a reasonable default since there may be conflicting configuration changes across version updates.

The settings are all stored in `%homepath%/.sleap` on Windows or `~/.sleap` on Linux/Mac. You can copy the contents of one folder from an older version to a newer one to transfer your settings.

I think we could at least have this documented on website, and perhaps in the future consider an option in the GUI itself to import settings from previous versions.

Cheers,

Talmo
